packages/requisitions/src/ResponseRequisition/DetailView/ContentArea.tsx

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import { DataTable, useTranslation } from '@openmsupply-client/common';
 import { useResponseLines, ResponseLineFragment } from '../api';
+import { RegexUtils } from '@openmsupply-client/common';
```

Comment:
:nit nicer to have from `import { RegexUtils } from '@common/utils';`

Reply:
These are now in the hooks, and I've just included the import with a large bunch of existing imports from `@openmsupply-client/common`. That okay?